src/pretix/control/templates/pretixcontrol/event/tickets.html

```diff
@@ -29,7 +29,7 @@ <h1>{% trans "Ticket download" %}</h1>
                     <div class="panel panel-default ticketoutput-panel">
                         <div class="panel-heading">
                             <a href="{% url "control:event.settings.tickets.preview" event=request.event.slug organizer=request.organizer.slug output=provider.identifier %}"
-                                    class="btn btn-default btn-sm pull-right {% if not provider.preview_allowed %}disabled{% endif %}"
+                                    class="btn btn-default btn-sm pull-right {% if not provider.preview_allowed or provider.download_handled_by_frontend %}disabled{% endif %}"
```

Comment:
Your PR still introduces ``preview_allowed`` (which is fine by me), so that one should be used here.